python/private/py_package.bzl

```diff
@@ -31,36 +31,62 @@ def _path_inside_wheel(input_file):
     return short_path
 
 def _py_package_impl(ctx):
-    inputs = depset(
-        transitive = [dep[DefaultInfo].data_runfiles.files for dep in ctx.attr.deps] +
-                     [dep[DefaultInfo].default_runfiles.files for dep in ctx.attr.deps],
-    )
-
     # TODO: '/' is wrong on windows, but the path separator is not available in starlark.
     # Fix this once ctx.configuration has directory separator information.
     packages = [p.replace(".", "/") for p in ctx.attr.packages]
-    if not packages:
-        filtered_inputs = inputs
+    exclude = [p.replace(".", "/") for p in ctx.attr.exclude]
+    input_files = depset(transitive = [dep[PyInfo].transitive_sources for dep in ctx.attr.deps] + [dep[DefaultInfo].default_runfiles.files for dep in ctx.attr.deps])
```

Comment:
Please reformat this line to make it shorter; it's almost two screen columns wide and hard to read in the diff.

Reply:
Fixed in 910a2b0.